<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>19</y>
        <width>571</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter client</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Code client :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_code">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_nom">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_prenom">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Né(e) le :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresse : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_adresse">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>371</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Code postal : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_cp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>N° de téléphone  :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_num_tel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>270</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>E-mail :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_email">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>300</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type du permis :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_permis_type">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>340</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>340</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Permis n° :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_permis_numero">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>340</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Délivré le :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>380</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Fin de validité :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pièce ID n° :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_cin">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>420</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>470</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>470</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_date_naissance">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_date_del_permis">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>380</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_datefindel_permis">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>380</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Modifier client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>531</width>
        <height>541</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier client</string>
      </property>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_nom2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_prenom2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Né(e) le :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresse : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_adresse2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>371</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Code postal : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_cp2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>N° de téléphone  :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_num_tel2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>E-mail :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_email2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_38">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type du permis :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_permis_type2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_39">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Permis n° :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_permis_numero2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>260</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Délivré le :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_42">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>300</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Fin de validité :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_43">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pièce ID n° :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_cin2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>340</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_date_naissance2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_date_del_permis2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>300</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_datefindel_permis2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>300</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_46">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier le client :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_code2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>651</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer client</string>
      </property>
      <widget class="QLabel" name="label_48">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Code du client à supprimer :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="client_code3">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Lister clients</string>
     </attribute>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Client :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_38">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>81</y>
        <width>681</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
